docs/sql-ref-ansi-compliance.md

```diff
@@ -404,6 +404,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>USING</td><td>reserved</td><td>strict-non-reserved</td><td>reserved</td></tr>
   <tr><td>VALUES</td><td>non-reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>VIEW</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
+  <tr><td>VIEWS</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
```

Comment:
Can you update `TableIdentifierParserSuite`, too?

Reply:
Sure, updated in c5570d249426a821896ab85e37bf2e0c1d9a39ee.